### DISCUSSION TITLE
Angular Support

### DISCUSSION THREAD

**Discussion by ch314s** [tailwindlabs/headlessui#1251] [General]
**HeadlessUI** is great for **VueJS** and **ReactJS**!

I often use it with **VueJS**.
If/How/When you are going to support **Angular**?

Best regards, 
**Chhai**

**Comment by RobinMalfait** [comment]
Hey! Thank you for your question!
Much appreciated! 🙏

We don't have any Angular support planned right now, so it might even be never.
